conf/solr/rmlog.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+find /var/log/tomcat7/* -mtime 7 -exec rm {} \;
```

Comment:
I tried this locally and looks like a plus was needed here? For reasons not entirely clear to me :P 

```suggestion
find /var/log/tomcat7/* -mtime +7 -exec rm {} \;
```

Reply:
It looks like `-mtime 7` matches files that are exactly 7 days old, while `-mtime +7` matches those that are greater than seven.  More details can be found [here](https://stackoverflow.com/questions/25599094/explaining-the-find-mtime-command#answer-25599430).